<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimplePhone</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_answer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bye">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Bye</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reject">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reject</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hold">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_transfer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_call">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
